ts/Series/Sankey/SankeySeries.ts

```diff
@@ -335,8 +335,16 @@ class SankeySeries extends ColumnSeries {
     }
 
     public drawDataLabels(): void {
-        ColumnSeries.prototype.drawDataLabels.call(this, this.points);
-        ColumnSeries.prototype.drawDataLabels.call(this, this.nodes);
+        for (const link of this.points) {
```

Comment:
This modification was supposed to be added to `OrganizationSeries.drawDataLabels` 😅 
Please restore the original Sankey method and move the correct class :)